vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -335,6 +335,11 @@ class RoomDetailFragment @Inject constructor(
             }
         }
 
+        // watch for permission changes concerning calls and enable/disable call buttons accordingly
+        roomDetailViewModel.selectSubscribe(RoomDetailViewState::isAllowedToManageWidgets, RoomDetailViewState::isAllowedToStartWebRTCCall) { _, _ ->
+            invalidateOptionsMenu()
```

Comment:
You are not observing the roomSummary anymore? I think it was useful to know the number of members

Reply:
`invalidateOptionsMenu()` is called in the main `invalidate` method, so thinking again about it, I think this `selectSubscribe` is not needed.

Reply:
Thanks for feedback, you were right, without the subscription it still works (I am new to the MvRx framework and didn't know about the "global" `invalidate()` method, I tested it with the following scenarios:
- Change `m.call.invite` power level
- Change "widget management" permission/power level
- removing members from a room with 3 participants and thus a user with no permission to manage widgets gaining permission to call

I added a commit to remove the unneeded state subscription, could also squash those into one if wanted.

Reply:
Thanks a lot for the update and for your tests!